docs/auth/federated-auth.md

```diff
@@ -223,80 +223,48 @@ import 'package:firebase_auth/firebase_auth.dart';
 
-* {iOS+ and Android}
-
-  On native platforms, a 3rd party library is required to both install the Twitter SDK and trigger the authentication flow.
-
-  Install the [`twitter_login`](https://pub.dev/packages/twitter_login) plugin:
-
-  ```yaml title="pubspec.yaml"
-  dependencies:
-    twitter_login: ^4.0.1
-  ```
+You also might need to request elevated [API access depending on your app](https://developer.twitter.com/en/portal/products/elevated).
 
-  Make sure to carefully go through the configuration steps of [`twitter_login`](https://pub.dev/packages/twitter_login) and register a callback URL at the [Twitter Developer Portal](https://developer.twitter.com/) with a matching URL scheme
+* {iOS+}
 
-  ```dart
-  import 'package:twitter_login/twitter_login.dart';
-
-  Future<UserCredential> signInWithTwitter() async {
-    // Create a TwitterLogin instance
-    final twitterLogin = new TwitterLogin(
-      apiKey: '<your consumer key>',
-      apiSecretKey:' <your consumer secret>',
-      redirectURI: '<your_scheme>://'
-    );
+  You need to configure custom URL scheme as [described in iOS guide step 1](https://firebase.google.com/docs/auth/ios/twitter-login#handle_the_sign-in_flow_with_the_firebase_sdk).
 
-    // Trigger the sign-in flow
-    final authResult = await twitterLogin.login();
-
-    // Create a credential from the access token
-    final twitterAuthCredential = TwitterAuthProvider.credential(
-      accessToken: authResult.authToken!,
-      secret: authResult.authTokenSecret!,
-    );
+* {Android}
 
-    // Once signed in, return the UserCredential
-    return await FirebaseAuth.instance.signInWithCredential(twitterAuthCredential);
-  }
-  ```
+  If you haven't yet specified your app's SHA-1 fingerprint, do so from the [Settings page](https://console.firebase.google.com/project/_/settings/general/) 
+  of the Firebase console. Refer to [Authenticating Your Client](https://developers.google.com/android/guides/client-auth) for details on how to get your app's SHA-1 fingerprint.
 
 * {Web}
 
-  On the web, the Twitter SDK provides support for automatically handling the authentication flow using the
-  Twitter application details provided on the Firebase console. Ensure that the callback URL in the Firebase console is added
-  as a callback URL in your Twitter application on their developer console.
-
-  For example:
-
-  Create a Twitter provider and provide the credential to the `signInWithPopup` method. This will trigger a new
-  window to appear prompting the user to sign-in to your Twitter application:
-
-  ```dart
-  Future<UserCredential> signInWithTwitter() async {
-    // Create a new provider
-    TwitterAuthProvider twitterProvider = TwitterAuthProvider();
+  Before you begin [configure Sign In with Apple](/docs/auth/web/apple#configure-sign-in-with-apple)
+  and [enable Apple as a sign-in provider](/docs/auth/web/apple#enable-apple-as-a-sign-in-provider).
```

Comment:
Is this correct?

Reply:
Good catch 😁 Nothing to do for Web so I forgot to remove the copy paste